### PR TITLE
Added correct example app name

### PR DESCRIPTION
**Pull request by thePianoKid**
## Summary

`CONTRIBUTING.md` had multiple references to this repo's example project. In `CONTRIBUTING.md`, the example project was supposed to belong in the `/example` directory, when it was actually in the `/example-app` directory. The `/example` directory does not exist. 

## Motivation

This is a tiny correction, but it adds a bit of unnecessary friction to the onboarding process :) 

## Testing

N/A - all changes were made to docs. None of the functionality was changed. 

- [ ] I tested this manually
- [ ] I added automated tests

## Documentation

Select one:

- [x] I have added relevant documentation for my changes.
- [ ] This PR does not result in any developer-facing changes.

### CODE REVIEW

**Comment by thePianoKid** [comment]
☝️ Bitrise is failing, but my change doesn't effect the functionality... am I doing something wrong?

**Comment by nazli-stripe** [comment]
@thePianoKid thanks for the PR! Bitrise requires admin approval to run the tests, which I just approved. It should pass as this is just a ReadMe change, after which I'll merge the PR!

**Review by nazli-stripe — APPROVED** [review]

**Pull request merged**
